applications/nrf_desktop/doc/swift_pair_app.rst

```diff
@@ -23,6 +23,9 @@ Module events
 Configuration
 *************
 
+To enable the module, use the :ref:`CONFIG_DESKTOP_SWIFT_PAIR <config_desktop_app_options>` Kconfig option.
```

Comment:
Should we globally switch to describing the Kconfig option that enables the module before describing the needed dependencies?

Reply:
We used to do it the other way round if we described multiple prerequisite steps that are needed

Reply:
I think it is good to first state which Kconfig option enables the module and then list anything else.